src/containers/ErrorMessages.js

```diff
@@ -13,14 +46,13 @@ export class ErrorMessages extends Component {
   componentWillUnmount() {
     this.unlisten();
   }
-
   render() {
     const { errors, dismissError } = this.props;
     return (
       <ToastNotificationList>
-        {[...new Set(errors.map(error => error.message))].map((error, index) => (
-          <ToastNotification key={index} onDismiss={() => dismissError(error)}>
-            <span>{error}</span>
+        {[...new Set(errors.map(error => errorMessage(error)))].map((error, index) => (
+          <ToastNotification key={index} onDismiss={() => dismissError(error.message)}>
+            <span>{error.dispalyMessage || error.message}</span>
```

Comment:
Typo: should be "displayMessage".

Reply:
This has been fixed in a different commint